### ISSUE TITLE
After downloading the files disappear last days. Windows 11 latest update Jan 2024.

### ISSUE THREAD

**Issue by OldBlogger01** [StefanLobbenmeier/youtube-dl-gui#105]
After downloading the files keep disappearing last days.
 Windows 11 latest update Jan 2024.
Also I see temporary folders in download location, that transform to files, if I use another way of download.
Version: 2.5.4
Windows 11 Pro
22621.3085
Windows Feature Experience Pack 1000.22684.1000.0
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/144936620/d534a192-a931-4ff3-97e8-e3db285a9c65)
Oups!! I was trying to copy the image of downloading and it worked fine again!!
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/144936620/21166ff6-b0c3-4105-a8fe-b0bfa99f342c)

**Comment by StefanLobbenmeier** [comment]
It might be the setting „prevent failing to save duplicate files“, that feature is a bit buggy for some circumstances

**Comment by OldBlogger01** [comment]
> It might be the setting „prevent failing to save duplicate files“, that feature is a bit buggy for some circumstances

Yes, I  think it is. It was one of the only thing I tried to change
